packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -57,26 +133,163 @@ export default defineComponent({
       )
     })
 
+    let isShowMore = computed(() => {
+      if (!balances.value.length) {
+        return
+      }
+
+      if (!state.searchQuery) {
+        return filteredBalanceList.value.length < balances.value.length
+      } else {
+        return filteredBalanceList.value.length < filteredArrayLength
+      }
+    })
+
+    const filteredBalanceList = computed(() => {
```

Comment:
this is still over complicated =

- filtered computed should only care about filtering 
- no need to create local vars from outer ones
- no need to care for balances being empty (see comp. getter)
- no need to store length of filtered arr